src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const longTermOfDays = 7;
+  const longRentalDiscount = 50;
+  const shortTermOfDays = 3;
+  const shortRentalDiscount = 20;
+
+  if (days >= longTermOfDays) {
```

Comment:
Prefer if with return over if else to simplify later conditions.
DON'T add else after if with return - the code after it won't be executed anyway